flag.go

```diff
@@ -578,8 +578,16 @@ func withEnvHint(envVar, str string) string {
 	return str + envText
 }
 
-func stringifyFlag(f Flag) string {
+func flagValue(f Flag) reflect.Value {
 	fv := reflect.ValueOf(f)
+	for fv.Kind() == reflect.Ptr {
+		fv = reflect.Indirect(fv)
+	}
+	return fv
```

Comment:
no need to test for the pointer, `Indirect` returns the `Value` if it is not a pointer. You can just do:

```go
return reflect.Indirect(reflect.ValueOf(f))
```